src/main/java/io/r2dbc/postgresql/UniformLoadBalancerConnectionStrategy.java

```diff
@@ -172,6 +173,7 @@ public synchronized boolean refresh(PostgresqlConnection controlConnection) {
                 }
             }
         }
+        controlConnection.close().block();
```

Comment:
Why do close the connection here if we want to use the same control connection?

Reply:
Just closing the blocked controlConnection, the Mono of it is still present